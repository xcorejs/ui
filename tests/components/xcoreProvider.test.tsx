import 'jest-styled-components';

import { XcoreProvider } from '@xcorejs/ui';
import React from 'react';
import renderer from 'react-test-renderer';

test('XcoreProvider component', () => {
  const component = renderer.create(
    <XcoreProvider />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
