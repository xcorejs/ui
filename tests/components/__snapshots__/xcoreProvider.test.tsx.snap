// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`XcoreProvider component 1`] = `null`;
